state/state.go

```diff
@@ -175,6 +173,7 @@ func MakeGenesisState(genDoc *types.GenesisDoc) (State, error) {
 		LastBlockID:     types.BlockID{},
 		LastBlockTime:   genDoc.GenesisTime,
 
+		NextValidators:              types.NewValidatorSet(validators).CopyIncrementAccum(1),
```

Comment:
Why we call CopyIncrementAccum here?

Reply:
Since it applies to the next height, it should have the accum bumped for the new proposal for that next height

Reply:
Maybe add comment `// since it applies to the next height, it should have the accum bumped for the new proposal for that next height`?